src/pages/MyTimePage/MyTimePage.tsx

```diff
@@ -55,6 +55,19 @@ export const MyTimePage = () => {
   const getDayComponent = (day: string) => {
     const colorIndex = -1;
     const freeHours = tasks[day]?.freeHrsOfDay;
+    
+    const today = new Date();
+    const dayOfMonth = today.getDate();
+    let suffix = "";
+    if (dayOfMonth === 1 || dayOfMonth === 21 || dayOfMonth === 31) {
+      suffix = "st";
+    } else if (dayOfMonth === 2 || dayOfMonth === 22) {
+      suffix = "nd";
+    } else if (dayOfMonth === 3 || dayOfMonth === 23) {
+      suffix = "rd";
+    } else {
+      suffix = "th";
```

Comment:
create a function in utils to get this suffix. Let's not crowd JSX file with these kind of functions

Reply:
Created function in utils.js as requested